src/lib/testHelpers/testRenderer.tsx

```diff
@@ -0,0 +1,23 @@
+// This helper is a custom implementation of
+// React Test Renderer that provides
+// Components a memory router as well as
+// Any additional providers that you specify.
+import React from 'react';
+import { ReactElement } from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import TestRenderer, {
+  ReactTestRenderer,
+  TestRendererOptions,
+} from 'react-test-renderer';
+
+function testRenderer(
+  nextElement: ReactElement,
+  options?: TestRendererOptions,
+): ReactTestRenderer {
+  return TestRenderer.create(
+    <MemoryRouter>{nextElement}</MemoryRouter>,
+    options,
+  );
+}
+
+export default testRenderer;
```

Comment:
Added a custom test renderer. This has the same API as React Test Renderer but takes in any number of providers to render the component with.